### PR TITLE
Document current TAC github process

### PR DESCRIPTION
**Pull request by dthaler**
Addresses issue #27 

Signed-off-by: Dave Thaler <dthaler@ntdev.microsoft.com>

### CODE REVIEW

File: tac-github-process.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+1. The chair checks the issues and PRs to decide which, if any,
+   should be on the next meeting agenda.
+2. All TAC members (not just the voting members) should have sent their
+   github ids to the chair and the Acting Secretary (Stephano) so any
+   TAC PRs can have all TAC members as reviewers.
+3. Once PRs are approved by two TAC members (including the author,
+   if a TAC member) with no change requests from any other reviewers,
+   then the "time bomb" label is added with a timeout of 1 week.
+4. An issue is merged once it's been open for at least a week and either
```
Comment (cetola) [inline tac-github-process.md:13]: Please change "An issue" to "A PR" or "A pull request".

**Comment by dthaler** [comment]
@cetola addressed feedback, please check

**Comment by dthaler** [comment]
Merging since PR has been open for a week and has 2 other approvers.

**Comment by dthaler** [comment]
Slated to be merged any time July 30th or later, if no objections raised by then.

**Comment by cetola** [comment]
Per today's TAC meeting these changes are approved and can be merged.

**Pull request merged**